tests/setTest.js

```diff
@@ -0,0 +1,64 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import Set from '../src/set'
+
+chai.use(chaiChange)
+
+describe('Set', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(Set).to.be.a('function')
+  })
+
+  context('add()', () => {
+    it('adds element to set', () => {
+      const mySet = new Set(['a', 'b', 3, 'zeze'])
+      console.log(mySet)
+
+      expect(() => mySet.add('foo'))
+        .to.alter(() => mySet.size(), { from: 4, to: 5 })
+    })
+  })
+
+  context('isEmpty()', () => {
+    it('checks if set is empty', () => {
+      const mySet = new Set([])
+      let EmptySet = mySet.isEmpty()
+      expect(EmptySet).to.deep.eql(true)
+      mySet.add('lemons')
+      let notemptySet = mySet.isEmpty()
+      expect(notemptySet).to.deep.equal(false)
+
+    })
+  })
+
+  context('contains()', () => {
+    it('returns true if set contains element we are passing', () => {
+      const mySet = new Set([1, 2, 3, 4])
+      let setContains = mySet.contains(3)
+      expect(setContains).to.deep.eql(true)
```

Comment:
if the function only returns a boolean it doesn't need to be deep equal. not sure if I'm reading this right